tests/common.bash

```diff
@@ -551,7 +551,7 @@ function install_nydus_snapshotter() {
 	version="${1}"
 
 	project="containerd/nydus-snapshotter"
-	tarball_name="nydus-snapshotter-${version}-$(${repo_root_dir}/tests/kata-arch.sh).tgz"
+	tarball_name="nydus-snapshotter-${version}-$(uname -s| tr A-Z a-z)-$(${repo_root_dir}/tests/kata-arch.sh).tar.gz"
```

Comment:
Do we need to use something like kata-arch.sh -g now we want amd64 rather than x86_64` in the tarball name?
```suggestion
	tarball_name="nydus-snapshotter-${version}-$(uname -s| tr A-Z a-z)-$(${repo_root_dir}/tests/kata-arch.sh -g).tar.gz"
```

Reply:
I second @stevenhorsman 's suggestion.

Reply:
Thanks @stevenhorsman. I have fixed it.